src/main/java/Ruby.java

```diff
@@ -0,0 +1,112 @@
+/**
+ * Main class for the chatbot named Ruby.
+ * Ruby assists users in recording and organizing their daily tasks.
+ * It interacts with users through the command line, processing commands to manage tasks.
+ */
+import java.util.Scanner;
```

Comment:
Great that you specificied java.util.Scanner instead of just importing java.util